src/connection.js

```diff
@@ -1057,6 +1101,41 @@ class Connection extends EventEmitter {
     }
   }
 
+  processLogin7KerberosResponse() {
+    if (this.loggedIn) {
+      return this.dispatchEvent('loggedIn');
```

Comment:
Maybe I'm missing something, but is this really the right location for this call? Where is `this.loggedIn` set to true for kerberos authentication? 🤔

Reply:
Ok, I think I figured out where this happens. 😓